examples/static-site-gen/platform/src/lib.rs

```diff
@@ -313,3 +326,17 @@ fn is_roc_code_block(cbk: &pulldown_cmark::CodeBlockKind) -> bool {
         }
     }
 }
+
+fn replace_code_with_static_file(code: &str, input_file: &Path) -> Option<String> {
+    let input_dir = input_file.parent()?;
+    let trimmed_code = code.trim();
+    if trimmed_code.len() <= 255 && trimmed_code.contains(".") {
```

Comment:
If you have your file deep in some nested folders it seems easy to exceed this 255 limit.
I thought some more about this and I do think we should opt for something like the `[filename.roc will be embedded here](path/filename.roc)` that Georges suggested. Because you can basically never trigger that by accident. Because you can not trigger it by accident we can provide nice error messages when the file can not be found, in contrast to the silent failure that could occur with the current code. For security reasons it would also be good to enforce that the file is located within the current directory (nested subdirectories are still allowed). Otherwise, there could be the risk that malicious user-generated md files are used to view the content of any file with read permissions on a server.